airbyte-integrations/connectors/source-shopify/metadata.yaml

```diff
@@ -75,6 +75,14 @@ data:
         scopedImpact:
           - scopeType: stream
             impactedScopes: ["countries"]
+      2.6.0:
+        message:
+          "The `ProductsGraphQL` and the `CustomerSavedSearch` streams are now deprecated and removed. Use the `Products` stream instead.
+          More details here: https://github.com/airbytehq/airbyte/pull/51037."
+        upgradeDeadline: "2025-02-15"
+        scopedImpact:
+          - scopeType: stream
+            impactedScopes: ["product_graph_ql", "customer_saved_search"]
```

Comment:
Is there a typo here? Should it be `products_graph_ql` instead of `product_graph_ql`?

Reply:
Yes, thanks. Fixed.